src/sidebar/store/modules/annotations.js

```diff
@@ -564,6 +564,7 @@ function savedAnnotations(state) {
  * @prop {(tags: string[]) => string[]} findIDsForTags
  * @prop {() => string[]} focusedAnnotations
  * @prop {() => string[]} highlightedAnnotations
+ * @prop {(tags: string) => boolean} isAnnotationFocused
```

Comment:
```suggestion
 * @prop {(tag: string) => boolean} isAnnotationFocused
```

Singular `tag` since we're talking about a single annotation here.